src/org/omegat/core/segmentation/SRX.java

```diff
@@ -110,40 +105,105 @@ public SRX copy() {
     }
 
     /**
-     * Saves segmentation rules into specified file.
+     * Saves segmentation rules into specified directory.
+     * Possibilities:
+     * -- if file does not exist : SRX format
+     * -- if conf file or srx file exists: owerwrite it
      */
-    public static void saveTo(SRX srx, File outFile) throws IOException {
+    public static void saveTo(SRX srx, File outDir) throws IOException {
+        File outFile = new File (outDir, SRX_SENTSEG);       
+        
         if (srx == null) {
             outFile.delete();
             return;
         }
+        
+        saveToSrx (srx, outFile);
+    }
+    
+    public static void saveToSrx(SRX srx, File outFile) throws IOException {        
+        ObjectFactory factory = new ObjectFactory();
+        Srx jaxbObject = factory.createSrx();
+        jaxbObject.setVersion("2.0");
+        jaxbObject.setHeader (factory.createHeader());
```

Comment:
Please clean up the spaces before the argument lists in this line and the following ones.